packages/create-plugin/templates/common/_package.json

```
@@ -23,7 +23,7 @@
     "@grafana/eslint-config": "^8.0.0",{{#if usePlaywright}}
     "@grafana/plugin-e2e": "^1.17.1",{{/if}}
     "@grafana/tsconfig": "^2.0.0",{{#if usePlaywright}}
-    "@playwright/test": "1.50.1",{{/if}}
+    "@playwright/test": "1.51.1",{{/if}}
```

I think we can put the caret back here now. I pinned it cos of this issue I couldn't figure out.

```suggestion
    "@playwright/test": "^1.51.1",{{/if}}
```